skulpt.py

```diff
@@ -175,7 +175,7 @@ def is64bit():
     return sys.maxsize > 2**32
 
 if sys.platform == "win32":
-    winbase = ".\\support\\d8\\x32"
+    winbase = ".\\support\\d8\\x32_"
```

Comment:
I see why you did this but, I think you should change this to `".\\support\\d8.exe"` and remove line 179 and 180. I don't see the need of this 32 bit check.